indico/modules/categories/models/categories.py

```diff
@@ -291,6 +297,11 @@ def display_tzinfo(self):
         """The tzinfo of the category or the one specified by the user."""
         return get_display_tz(self, as_timezone=True)
 
+    def can_propose_events(self, user):
+        """Check whether the user can propose move requests to the category."""
+        return self.event_requires_approval or (
+            user and self.can_manage(user, permission='event_move_request')) or self.can_create_events(user)
```

Comment:
Shouldn't it return false if;

- `user` is none or
- the user can create events (i guess we'll never show the approval flow in that case anyway?)

Reply:
Yes, `event_requires_approval` allows anyone even though the user shouldn't be none. But as for the second option, I made this method a bit more generic or hierarchical as in "if you can create events you are also allowed to propose them". But it's a fair change.